### DISCUSSION TITLE
Source JDBC Database hashing columns before transferring them

### DISCUSSION THREAD

**Discussion by pschrammel** [airbytehq/airbyte#28385] [Connector Questions]
Hi,
I'm trying to hash some DB columns before transfering them. But I don't know where to start. I looked at this article which describes the right thing on a wrong source: https://airbyte.com/blog/etlt-gdpr-compliance . 
Does anybody have an example for postgres databases?

**Comment by marcosmarxm** [comment]
The article is limited to source databases where you can control the fields you're going to ingest. For databases connectors I strongly recommend to you create a view where you can filter, remove or hash the source data for the purpose you want.

**Reply by pschrammel** [reply]
thx, in my case I can't modify the external DB (other team, ....).

**Reply by marcosmarxm** [reply]
I see, unfortunately the actual connector doesn't allow you to apply any transformation to your data.
Also Airbyte allow you to unselect some columns maybe this can be an option for you to not ingest the data at all.

[a comment by an automated account is not shown]
